blog/tidal_profile_ndr_brt.md

```diff
@@ -0,0 +1,67 @@
+
+**What functions and coding approaches do you like to use?**
+I'm a huge fan of `superimpose` (used with the `si` shortand), especially mixed with `hurry`, in my "single sample runs" I create layers of the same sample playing at different speed/density to create rhythm and melodic patterns, I often try to escape from this mindset but at the end I fall into it most of the times.
+
+**Do you use Tidal with other tools / environments?**
+I tried it to control some drum machines circuit bent by me but at the end I find the hardware overcomplicated and I prefer to play soft-synths, especially Supercollider: everything in a box and controllable with the keyboard.
+I used Tidal also to [draw stuff with p5 during some sessions](https://www.youtube.com/watch?v=lbUCSVC4GPs).
+
+## Tidal Contributions  
+
+**How do you contribute to Tidal Cycles? What have you worked on?**
+I learned Haskell only to contribute to Tidal, I'm passionate about reading code and get the insights of the software, on the main codebase I solved some bugs and added some feature mainly in the mini-notation section. I also took care of the migration from Travis CI to GitHub Actions.
+At a certain point I noticed that the Atom Plugin was practically unmaintained so I proposed to be its maintainer, and I brought it back on track. Now Atom has been disbanded but luckily the Pulsar community is vibrant and the Tidal plugin is already fully compatible with it. It was a pretty satisfying migration.
+
+**What motivates you to work on Tidal?**
+Not to be selfish but most of the work I did had direct impact on the use I'm doing of Tidal, I guess because it's easy to contribute when you know why something needs to be improved/fixed.
+
+
+## Music  
+
+**Tell us about your livecoding music.**
+Well, most of the time it is noisy, sometimes mellow, always not danceable.
+
+**How has your music evolved since you have been livecoding?**
+For sure it changed, I'm not sure it "evolved", sometime I think I was more creative when I was learning how to use the instrument, now it's easier to get into the loop of being repetitive.
+
+**What samples or instruments do you like to work with?**
+I try not to use the default samples nor the default synths, I sometimes write my own synths, sometimes I record my own samples or I get them from various sources  
+
+**What projects are you currently working on or planning? What's next?**
+I recently finished a record that was a collaboration with Naotodate, that's a non-livecoding noise friend from Italy. Now I'm working to another collaboration record, this time with ETOL, an amazing italian livecoder. To be fair the project it is still in an embryonic state.
+
+
+### Links to your music / recorded livecoding sessions:
+- [Bandcamp](https://umanoudito.bandcamp.com/)
+- [YouTube](https://www.youtube.com/@ndrbrt) (not only livecoding)
```

Comment:
You could add several links to specific videos that you really like as a nested list under YouTube.